### DISCUSSION TITLE
How to Make Open-WebUI Accessible on a the Internet

### DISCUSSION THREAD

**Discussion by abcde89525** [open-webui/open-webui#2641] [General]
I would like to ask how to set up Open-WebUI so that I can use it on my mobile phone. I really like this project, but I've spent several hours trying to find a way to use it on my phone within the same Wi-Fi environment without success. I can only access the website on the desktop where I run Open-WebUI using 127.0.0.1:8080.

**Comment by cokice** [comment]
nginx反向代理

**Comment by delfireinoso** [comment]
you can use a SSH tunnel, I use http://zrok.io to create a tunnel to localhost:port 

You can establish a permanent conection with the reserve command or running it on a docker image

**Reply by yanggun7201** [reply]
You can use ngrok
